src/applications/mhv-medical-records/util/txtHelpers/blueButton.js

```diff
@@ -42,12 +47,22 @@ ${txtLineDotted}
   4. Allergies
   5. Health Conditions
   6. Vitals
+  7. medications,
+  8. appointments,
+  9. demographics,
+  10. militaryService,
+  11. accountSummary,
```

Comment:
Capitalize and space these appropriately, like the ones above.

Reply:
Done